diff -ru MUMPS_5.9.0/src/cfac_b.F MUMPS/src/cfac_b.F
--- MUMPS_5.9.0/src/cfac_b.F	2026-04-28 15:06:26.000000000 +0200
+++ MUMPS/src/cfac_b.F	2026-05-19 14:56:33.796076002 +0200
@@ -38,7 +38,7 @@
      &                     , CMUMPS_BUF_DEALL_MAX_ARRAY
       USE CMUMPS_FAC_S_IS_POINTERS_M, ONLY : CMUMPS_S_IS_POINTERS_T
       USE MUMPS_PIVNUL_MOD, ONLY : PIVNUL_LIST_STRUCT_T
-      USE OMP_LIB
+!$      USE OMP_LIB
       USE MUMPS_TPS_M
       USE CMUMPS_TPS_M
       USE CMUMPS_FAC_OMP_M
diff -ru MUMPS_5.9.0/src/cfac_compact_factors_m.F MUMPS/src/cfac_compact_factors_m.F
--- MUMPS_5.9.0/src/cfac_compact_factors_m.F	2026-04-28 15:06:28.000000000 +0200
+++ MUMPS/src/cfac_compact_factors_m.F	2026-05-19 14:56:33.796358527 +0200
@@ -19,7 +19,7 @@
      &      WK_USER_PROVIDED, S, KEEP, KEEP8, INFO, MYID, ICNTL,
      &      PROK, MP, CMUMPS_LBUFR_BYTES8, CMUMPS_LBUF8, 
      &      LIWK, LIWK8 )
-      USE OMP_LIB
+!$      USE OMP_LIB
       USE CMUMPS_DYNAMIC_MEMORY_M, ONLY : CMUMPS_DM_FREE_S_WK
 C
 C Purpose
diff -ru MUMPS_5.9.0/src/cfac_par_m.F MUMPS/src/cfac_par_m.F
--- MUMPS_5.9.0/src/cfac_par_m.F	2026-04-28 15:06:26.000000000 +0200
+++ MUMPS/src/cfac_par_m.F	2026-05-19 14:56:33.796507518 +0200
@@ -50,7 +50,7 @@
       USE CMUMPS_FAC2_LDLT_M
       USE CMUMPS_FAC1_LU_M
       USE CMUMPS_FAC2_LU_M
-      USE OMP_LIB
+!$      USE OMP_LIB
       USE MUMPS_TPS_M
       USE CMUMPS_TPS_M
       USE MUMPS_INTR_TYPES, ONLY : MUMPS_ROOT_STRUC
diff -ru MUMPS_5.9.0/src/csol_omp_m.F MUMPS/src/csol_omp_m.F
--- MUMPS_5.9.0/src/csol_omp_m.F	2026-04-28 15:06:25.000000000 +0200
+++ MUMPS/src/csol_omp_m.F	2026-05-19 14:57:57.684749493 +0200
@@ -324,7 +324,7 @@
      &  PERM_L0_OMP, PTR_LEAFS_L0_OMP, L0_OMP_MAPPING, LL0_OMP_MAPPING,
      &  L0_OMP_FACTORS, LL0_OMP_FACTORS )
       USE CMUMPS_INTR_TYPES, ONLY : CMUMPS_L0OMPFAC_T
-      USE OMP_LIB
+!$      USE OMP_LIB
       USE MUMPS_L0_OMP_M, ONLY : THREAD_ID
       IMPLICIT NONE
       INTEGER, INTENT( in ) :: N, MTYPE, NRHS, SLAVEF, LIW
diff -ru MUMPS_5.9.0/src/dfac_b.F MUMPS/src/dfac_b.F
--- MUMPS_5.9.0/src/dfac_b.F	2026-04-28 15:06:25.000000000 +0200
+++ MUMPS/src/dfac_b.F	2026-05-19 14:56:33.797152549 +0200
@@ -38,7 +38,7 @@
      &                     , DMUMPS_BUF_DEALL_MAX_ARRAY
       USE DMUMPS_FAC_S_IS_POINTERS_M, ONLY : DMUMPS_S_IS_POINTERS_T
       USE MUMPS_PIVNUL_MOD, ONLY : PIVNUL_LIST_STRUCT_T
-      USE OMP_LIB
+!$      USE OMP_LIB
       USE MUMPS_TPS_M
       USE DMUMPS_TPS_M
       USE DMUMPS_FAC_OMP_M
diff -ru MUMPS_5.9.0/src/dfac_compact_factors_m.F MUMPS/src/dfac_compact_factors_m.F
--- MUMPS_5.9.0/src/dfac_compact_factors_m.F	2026-04-28 15:06:27.000000000 +0200
+++ MUMPS/src/dfac_compact_factors_m.F	2026-05-19 14:56:33.797395206 +0200
@@ -19,7 +19,7 @@
      &      WK_USER_PROVIDED, S, KEEP, KEEP8, INFO, MYID, ICNTL,
      &      PROK, MP, DMUMPS_LBUFR_BYTES8, DMUMPS_LBUF8, 
      &      LIWK, LIWK8 )
-      USE OMP_LIB
+!$      USE OMP_LIB
       USE DMUMPS_DYNAMIC_MEMORY_M, ONLY : DMUMPS_DM_FREE_S_WK
 C
 C Purpose
diff -ru MUMPS_5.9.0/src/dfac_par_m.F MUMPS/src/dfac_par_m.F
--- MUMPS_5.9.0/src/dfac_par_m.F	2026-04-28 15:06:25.000000000 +0200
+++ MUMPS/src/dfac_par_m.F	2026-05-19 14:56:33.797520163 +0200
@@ -50,7 +50,7 @@
       USE DMUMPS_FAC2_LDLT_M
       USE DMUMPS_FAC1_LU_M
       USE DMUMPS_FAC2_LU_M
-      USE OMP_LIB
+!$      USE OMP_LIB
       USE MUMPS_TPS_M
       USE DMUMPS_TPS_M
       USE MUMPS_INTR_TYPES, ONLY : MUMPS_ROOT_STRUC
diff -ru MUMPS_5.9.0/src/dsol_omp_m.F MUMPS/src/dsol_omp_m.F
--- MUMPS_5.9.0/src/dsol_omp_m.F	2026-04-28 15:06:24.000000000 +0200
+++ MUMPS/src/dsol_omp_m.F	2026-05-19 14:58:16.177971436 +0200
@@ -324,7 +324,7 @@
      &  PERM_L0_OMP, PTR_LEAFS_L0_OMP, L0_OMP_MAPPING, LL0_OMP_MAPPING,
      &  L0_OMP_FACTORS, LL0_OMP_FACTORS )
       USE DMUMPS_INTR_TYPES, ONLY : DMUMPS_L0OMPFAC_T
-      USE OMP_LIB
+!$      USE OMP_LIB
       USE MUMPS_L0_OMP_M, ONLY : THREAD_ID
       IMPLICIT NONE
       INTEGER, INTENT( in ) :: N, MTYPE, NRHS, SLAVEF, LIW
diff -ru MUMPS_5.9.0/src/sfac_b.F MUMPS/src/sfac_b.F
--- MUMPS_5.9.0/src/sfac_b.F	2026-04-28 15:06:24.000000000 +0200
+++ MUMPS/src/sfac_b.F	2026-05-19 14:56:33.798164911 +0200
@@ -38,7 +38,7 @@
      &                     , SMUMPS_BUF_DEALL_MAX_ARRAY
       USE SMUMPS_FAC_S_IS_POINTERS_M, ONLY : SMUMPS_S_IS_POINTERS_T
       USE MUMPS_PIVNUL_MOD, ONLY : PIVNUL_LIST_STRUCT_T
-      USE OMP_LIB
+!$      USE OMP_LIB
       USE MUMPS_TPS_M
       USE SMUMPS_TPS_M
       USE SMUMPS_FAC_OMP_M
diff -ru MUMPS_5.9.0/src/sfac_compact_factors_m.F MUMPS/src/sfac_compact_factors_m.F
--- MUMPS_5.9.0/src/sfac_compact_factors_m.F	2026-04-28 15:06:27.000000000 +0200
+++ MUMPS/src/sfac_compact_factors_m.F	2026-05-19 14:56:33.798415085 +0200
@@ -19,7 +19,7 @@
      &      WK_USER_PROVIDED, S, KEEP, KEEP8, INFO, MYID, ICNTL,
      &      PROK, MP, SMUMPS_LBUFR_BYTES8, SMUMPS_LBUF8, 
      &      LIWK, LIWK8 )
-      USE OMP_LIB
+!$      USE OMP_LIB
       USE SMUMPS_DYNAMIC_MEMORY_M, ONLY : SMUMPS_DM_FREE_S_WK
 C
 C Purpose
diff -ru MUMPS_5.9.0/src/sfac_par_m.F MUMPS/src/sfac_par_m.F
--- MUMPS_5.9.0/src/sfac_par_m.F	2026-04-28 15:06:24.000000000 +0200
+++ MUMPS/src/sfac_par_m.F	2026-05-19 14:56:33.798543874 +0200
@@ -50,7 +50,7 @@
       USE SMUMPS_FAC2_LDLT_M
       USE SMUMPS_FAC1_LU_M
       USE SMUMPS_FAC2_LU_M
-      USE OMP_LIB
+!$      USE OMP_LIB
       USE MUMPS_TPS_M
       USE SMUMPS_TPS_M
       USE MUMPS_INTR_TYPES, ONLY : MUMPS_ROOT_STRUC
diff -ru MUMPS_5.9.0/src/ssol_omp_m.F MUMPS/src/ssol_omp_m.F
--- MUMPS_5.9.0/src/ssol_omp_m.F	2026-04-28 15:06:23.000000000 +0200
+++ MUMPS/src/ssol_omp_m.F	2026-05-19 14:58:20.311279884 +0200
@@ -324,7 +324,7 @@
      &  PERM_L0_OMP, PTR_LEAFS_L0_OMP, L0_OMP_MAPPING, LL0_OMP_MAPPING,
      &  L0_OMP_FACTORS, LL0_OMP_FACTORS )
       USE SMUMPS_INTR_TYPES, ONLY : SMUMPS_L0OMPFAC_T
-      USE OMP_LIB
+!$      USE OMP_LIB
       USE MUMPS_L0_OMP_M, ONLY : THREAD_ID
       IMPLICIT NONE
       INTEGER, INTENT( in ) :: N, MTYPE, NRHS, SLAVEF, LIW
diff -ru MUMPS_5.9.0/src/zfac_b.F MUMPS/src/zfac_b.F
--- MUMPS_5.9.0/src/zfac_b.F	2026-04-28 15:06:27.000000000 +0200
+++ MUMPS/src/zfac_b.F	2026-05-19 14:56:33.799159419 +0200
@@ -38,7 +38,7 @@
      &                     , ZMUMPS_BUF_DEALL_MAX_ARRAY
       USE ZMUMPS_FAC_S_IS_POINTERS_M, ONLY : ZMUMPS_S_IS_POINTERS_T
       USE MUMPS_PIVNUL_MOD, ONLY : PIVNUL_LIST_STRUCT_T
-      USE OMP_LIB
+!$      USE OMP_LIB
       USE MUMPS_TPS_M
       USE ZMUMPS_TPS_M
       USE ZMUMPS_FAC_OMP_M
diff -ru MUMPS_5.9.0/src/zfac_compact_factors_m.F MUMPS/src/zfac_compact_factors_m.F
--- MUMPS_5.9.0/src/zfac_compact_factors_m.F	2026-04-28 15:06:28.000000000 +0200
+++ MUMPS/src/zfac_compact_factors_m.F	2026-05-19 14:56:33.799398439 +0200
@@ -19,7 +19,7 @@
      &      WK_USER_PROVIDED, S, KEEP, KEEP8, INFO, MYID, ICNTL,
      &      PROK, MP, ZMUMPS_LBUFR_BYTES8, ZMUMPS_LBUF8, 
      &      LIWK, LIWK8 )
-      USE OMP_LIB
+!$      USE OMP_LIB
       USE ZMUMPS_DYNAMIC_MEMORY_M, ONLY : ZMUMPS_DM_FREE_S_WK
 C
 C Purpose
diff -ru MUMPS_5.9.0/src/zfac_par_m.F MUMPS/src/zfac_par_m.F
--- MUMPS_5.9.0/src/zfac_par_m.F	2026-04-28 15:06:27.000000000 +0200
+++ MUMPS/src/zfac_par_m.F	2026-05-19 14:56:33.799525861 +0200
@@ -50,7 +50,7 @@
       USE ZMUMPS_FAC2_LDLT_M
       USE ZMUMPS_FAC1_LU_M
       USE ZMUMPS_FAC2_LU_M
-      USE OMP_LIB
+!$      USE OMP_LIB
       USE MUMPS_TPS_M
       USE ZMUMPS_TPS_M
       USE MUMPS_INTR_TYPES, ONLY : MUMPS_ROOT_STRUC
diff -ru MUMPS_5.9.0/src/zsol_omp_m.F MUMPS/src/zsol_omp_m.F
--- MUMPS_5.9.0/src/zsol_omp_m.F	2026-04-28 15:06:26.000000000 +0200
+++ MUMPS/src/zsol_omp_m.F	2026-05-19 14:58:09.501344982 +0200
@@ -324,7 +324,7 @@
      &  PERM_L0_OMP, PTR_LEAFS_L0_OMP, L0_OMP_MAPPING, LL0_OMP_MAPPING,
      &  L0_OMP_FACTORS, LL0_OMP_FACTORS )
       USE ZMUMPS_INTR_TYPES, ONLY : ZMUMPS_L0OMPFAC_T
-      USE OMP_LIB
+!$      USE OMP_LIB
       USE MUMPS_L0_OMP_M, ONLY : THREAD_ID
       IMPLICIT NONE
       INTEGER, INTENT( in ) :: N, MTYPE, NRHS, SLAVEF, LIW
